docs/source/drivers-clients/http-client-server-api.md

```diff
@@ -1,11 +1,12 @@
 # The HTTP Client-Server API
 
-When you start Bigchaindb using `bigchaindb start`, an HTTP API is exposed at:
+Note: The HTTP client-server API is currently quite rudimentary. For example, there is no ability to do complex queries using the HTTP API. We plan to add querying capabilities in the future. If you want to build a full-featured proof-of-concept, we suggest you use [the Python server API](../nodes/python-server-api-examples.html) for now.
 
-- [http://localhost:9984/api/v1/](http://localhost:9984/api/v1/)
+When you start Bigchaindb using `bigchaindb start`, an HTTP API is exposed at:
 
+[http://localhost:9984/api/v1/](http://localhost:9984/api/v1/)
 
-Please note that for security reasons the server binds to `localhost:9984`.
+For security reasons, the server binds to `localhost:9984`.
```

Comment:
There may also be reasons to bind it to `0.0.0.0` in development (we do this in docker). We only need to change the api endpoint in the configuration file